draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,19 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint SHOULD elect to limit the number of outstanding RETIRE_CONNECTION_ID
+frames to bound the necessary state. In order to allow a peer to retire all
+previously issued connection IDs, the limit on the number of outstanding
+RETIRE_CONNECTION_IDs SHOULD be at least the active_connection_id_limit.  If the
+number of outstanding RETIRE_CONNECTION_IDs exceeds twice the
+active_connection_id_limit, the connection MAY be closed with a connection error
+of type CONNECTION_ID_LIMIT_ERROR.
+
+Endpoints SHOULD NOT issue updates of the Retire Prior To field before receiving
+RETIRE_CONNECTION_ID frames for the previous update of Retire Prior To. Doing so
+may result in the peer being unwilling to immediately retire all the connection
+IDs, if retiring them would cause it to exceed its limit on outstanding
+RETIRE_CONNECTION_ID frames, as described above.
```

Comment:
```suggestion
Endpoints SHOULD NOT issue updates of the Retire Prior To field before receiving
RETIRE_CONNECTION_ID frames for the previous update of Retire Prior To.
```
Rather than explain this and get the numbers right, just leave the recommendation.